jsapp/js/components/common/icon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+// Using this type ensures we only have existing icon names
+import {IconName} from '../../../fonts/k-icons'
+import './icon.scss'
+
+/**
+ * Check out `icon.scss` file for exact pixel values.
+ */
+export type IconSize = 'xs' | 's' | 'm' | 'l' | 'xl'
+
+const DefaultSize = 's'
+
+type IconProps = {
+  name: IconName
+  size?: IconSize
+  classNames?: string[]
```

Comment:
There's a lot of places in the code where we add `onChange` to icons (`mediaCell.es6` for example). Should we add that to this component or  should we just surround each case of this with a wrapper button?

Reply:
Yeah I think this is a better way. A clickable icon just doesn't feel right when I think about it 🤢 😉

Reply:
But of course if we find any property that would be needed, we will be adding it when the moment comes :)